### PR TITLE
Routes for comments

### PR DESCRIPTION
**Pull request by emcoding**
Redirect not working; routing not working
- If I add status_updates_controller to an additional folder, I am effectively namespacing. I didn't want that for the status_updates_controller and especially its views, so->
- I added only the comments_controller to the status_updates_folder. To no effect: it is never fired, and the PATH_PARENTS array stays empty for ever.

So, that is obviously not working.
If I add the status_updates_controller to the folder as well, I am back to the weird status_updates_status_update_path 's. And in the end, it won't find the show-view AND it won't let me add an additional view/status_updates/ -folder, because that is the name of the folder where the view currently lives.

I tried all kind of things, but I am running around in circles here.

### CODE REVIEW

File: app/controllers/status_updates_controller.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 class StatusUpdatesController < ApplicationController
+  # I think I don't need this; it doesn't make a difference anyway
```
Comment (michaelem) [inline app/controllers/status_updates_controller.rb]: yep, not needed here

File: app/views/status_updates/show.html.slim [hunk 2]
```diff
@@ -12,6 +12,30 @@ p
 .well.status_update
   = render_markdown(@status_update.content).html_safe
 
+h2#comments.header
+  = icon('comments')
+  span Comments
+
+- if current_user
+  = simple_form_for Comment.new(commentable: @status_update), url: comments_path do |f|
```
Comment (michaelem) [inline app/views/status_updates/show.html.slim]: That is the reason why you don't see any effect from the new controller: this is still posting to the old one. You could use something like: 'url: status_updates_comments_path'.

File: app/controllers/status_updates/comments_controller.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class StatusUpdates::CommentsController < CommentsController
+  PATH_PARENTS = [:status_updates]
```
Comment (michaelem) [inline app/controllers/status_updates/comments_controller.rb]: I think for the redirect you want you actually need to override the entire create method.

**Comment by emcoding** [comment]
@michaelem Could you check again please? This is working.

**Comment by michaelem** [comment]
looks good 🎉🎉🎉

**Pull request closed without merging**